android/assets/jsons/translations/Simplified_Chinese.properties

```
@@ -803,27 +799,25 @@ Your city [cityName] can bombard the enemy! = 你的城市[cityName]可以轰击
 [amount] of your cities can bombard the enemy! = 你有 [amount] 座城市可以轰击敌人！
 [amount] enemy units were spotted near our territory = 我们的领土附近发现了 [amount] 个敌方单位
 [amount] enemy units were spotted in our territory = 我们的领土内发现了 [amount] 个敌方单位
-A(n) [nukeType] exploded in our territory! = 一颗[nukeType]在我们的领土上爆炸了！
+A(n) [nukeType] exploded in our territory! = 一颗[nukeType]在我方领土上爆炸了！
 After being hit by our [nukeType], [civName] has declared war on us! = 在被我们的[nukeType]打击后，[civName]向我们宣战了！
-The civilization of [civName] has been destroyed! = [civName]文明已经灭亡！
-The City-State of [name] has been destroyed! = [name]城邦已经灭亡！
+The civilization of [civName] has been destroyed! = 文明[civName]消绝了！
+The City-State of [name] has been destroyed! = 城邦[name]覆灭了！
 Your [ourUnit] captured an enemy [theirUnit]! = 我们的[ourUnit]俘虏了敌方的[theirUnit]！
 Your [ourUnit] plundered [amount] [Stat] from [theirUnit] = 我们的[ourUnit]从敌方的[theirUnit]掠夺了 [amount] [Stat]！
 We have captured a barbarian encampment and recovered [goldAmount] gold! = 我们摧毁蛮族营地并获得了 [goldAmount] 金钱！
-An enemy [unitType] has joined us! = 一个敌方[unitType]加入了我们！
- # Requires translation!
-After an unknown civilization entered the [eraName], we have recruited [spyName] as a spy! = 
- # Requires translation!
-We have recruited [spyName] as a spy! = 
+An enemy [unitType] has joined us! = 敌方的[unitType]被我们成功策反！
```

```suggestion
An enemy [unitType] has joined us! = 一个敌方[unitType]加入了我们！
```
可能原翻译较合适。因为这条是在出间谍之前就有这条的，有时德国打蛮族也会出